Sources/Interaction/Manipulators/MouseRangeManipulator/index.js

```diff
@@ -9,6 +9,10 @@ function vtkMouseRangeManipulator(publicAPI, model) {
   // Set our className
   model.classHierarchy.push('vtkMouseRangeManipulator');
 
+  // Set a default value so that race conditions will not lead to
+  // attempting to access properties of an undefined model property
+  model.previousPosition = { x: 0, y: 0 };
```

Comment:
shouldn't x and y be undefined (or null) ?  because it is not quite true that they are at `(0,0)` at default

Reply:
If I set them to `null` then I also need to add checks on `onPointerLockMove` to avoid it using the value to perform math operations on it.

Honestly this doesn't matter because the value will be replaced with the correct one as soon the `mouseDown` method is called anyways.

Reply:
If you prefer I can add an early return on `onPointerLockMove` right before we attempt to access x/y, this will prevent the `onMouseMove` event from firing then.

Reply:
I just went ahead and made the change as it looks cleaner

Reply:
I think that makes more sense to me. Thanks !